<?xml version="1.0" encoding="utf-8"?>
<launch>
	<arg name="robot" default="robot" />
	<arg name="odom_topic" default="/robot/husky_velocity_controller/odom" />
	<arg name="pointcloud_topic" default="/robot/lidar_crop" />
	<arg name="base_frame" default="/base_link"/>
	<arg name="odom_frame" default="/odom"/>
	<arg name="map_frame" default="/map"/>

	<include file="$(find gmapping)/launch/pcToLaser_360.launch">
	</include>

	<include file="$(find mapping)/launch/pub_tf.launch">
		<!-- <arg name="robot" value="$(arg robot)" /> -->
	</include>

	<include file="$(find gmapping)/launch/slam_gmapping_pr2.launch">
		<arg name="base_frame" value="$(arg base_frame)"/>
		<arg name="map_frame" value="$(arg map_frame)"/>
		<arg name="odom_frame" value="$(arg odom_frame)"/>
		<arg name="scan" value="/robot/RL/scan"/>
	</include>

</launch>
